hackerforms/__init__.py

```diff
@@ -6,11 +6,13 @@
 from .generated.outputs import *
 from .generated.actions import *
 from .parameters import url_params
-
+from .version import __version__, check_version as _check_version
 from .socket import initialize as _initialize_socket
 from .overloads import initialize as _initialize_overloads
 
-
 if os.getenv("ENV") != "test":
+    _check_version()
     _initialize_socket()
     _initialize_overloads()
+
+del _check_version
```

Comment:
era bom por del dos outros tbm
```suggestion
del _check_version
def _initialize_socket
del _initialize_overloads
```